sklearn/utils/tests/test_cython_blas.py

```diff
@@ -17,10 +17,12 @@
 from sklearn.utils._cython_blas import RowMajor, ColMajor
 from sklearn.utils._cython_blas import Trans, NoTrans
 
-cython = pytest.importorskip("cython")
+
+def _numpy_to_cython():
+    cython = pytest.importorskip("cython")
+    return {np.float32: cython.float, np.float64: cython.double}
```

Comment:
You could make this take "dtype" as input and return the cython dtype directly, since it's how it is mostly used.

Reply:
Absolutely